demo/c/README.md

```diff
@@ -1,69 +1,107 @@
-# Compatibility
+# C Demos
 
-You need a C99-compatible compiler to build these demos. The microphone based demo can only run on Linux-based systems
-such as `Ubuntu`, `Raspberry Pi`, and `BeagleBone` as it depends on `ALSA`.
+## Compatibility
 
-# Usage
+You need a C99-compatible compiler to build these demos. The microphone based demo requires `miniaudio`
+for accessing microphone audio data.
 
 ## Microphone Demo
 
-Compile by executing the following command from the root of the repository
+## Requirements
+
+The microphone demo relies on `miniaudio`. To get `miniaudio` as submodule:
+
+```console
+git submodule update --init --recursive
+```
+
+## Build
+
+### Linux, macOS, Raspberry Pi
 
 ```console
-gcc -std=c99 -O3 -o demo/c/porcupine_demo_mic -I include/ demo/c/porcupine_demo_mic.c -ldl -lasound
+gcc -std=c99 -O3 -o demo/c/porcupine_demo_mic -I include/ demo/c/porcupine_demo_mic.c -ldl -lpthread -lm
 ```
 
-Running the executable without any commandline arguments prints the usage info to the console as below
+### Windows
 
 ```console
-$ ./demo/c/porcupine_demo_mic 
-usage : ./demo/c/porcupine_demo_mic library_path model_path keyword_path sensitivity input_audio_device
+gcc -std=c99 -O3 -o demo/c/porcupine_demo_mic -I include/ demo/c/porcupine_demo_mic.c
```

Comment:
I think we should mention that we tried it with MingW. One question that would come up is how you got GCC on Windows.